### PR TITLE
updates for issue 8006

### PR DESCRIPTION
**Pull request by shortpatti**
Docs: Change titles of collectors to "X monitoring with Netdata"

### CODE REVIEW

File: collectors/charts.d.plugin/apcupsd/README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# apcupsd
+# APCUPSD monitoring with Netdata
```
Comment (ilyam8) [inline collectors/charts.d.plugin/apcupsd/README.md]: `APCUPSD` is a tool 

> https://wiki.debian.org/apcupsd

Module uses it to monitors different APC UPS models.

File: collectors/charts.d.plugin/libreswan/README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
-# libreswan
+# LibreSwan monitoring with Netdata
```
Comment (ilyam8) [inline collectors/charts.d.plugin/libreswan/README.md]: `LibreSwan` is IPsec VPN implementation

> https://libreswan.org/

Modules monitors Libreswan IPsec Tunnels

File: collectors/charts.d.plugin/nut/README.md [hunk 3]
```diff
@@ -1,6 +1,6 @@
-# nut
+# NUT monitoring with Netdata
```
Comment (ilyam8) [inline collectors/charts.d.plugin/nut/README.md]: NUT is a suite of tool used by the module to get UPS statistics

> https://en.wikipedia.org/wiki/Network_UPS_Tools

Module monitors UPS

File: collectors/charts.d.plugin/sensors/README.md [hunk 4]
```diff
@@ -1,4 +1,4 @@
-# sensors
+# Sensors monitoring with Netdata
```
Comment (ilyam8) [inline collectors/charts.d.plugin/sensors/README.md]: I think we could be more specific here and use `Linux Machines Sensors` (modules monitors only linux machine sensors)

**Comment by ilyam8** [comment]
hi @shortpatti 

i started to review but lets do it that way

Please use https://docs.netdata.cloud/collectors/collectors/ page as a reference what module monitors and fix rest headers. 


We change h1 header from `ModuleName` to `<MonitoredService> monitoring with netdata`.

As you may see `ModuleName` != `<MonitoredService>`.

File: collectors/python.d.plugin/unbound/README.md [hunk 5]
```diff
@@ -0,0 +1,114 @@
+# Unbound monitoring with Netdata
```
Comment (joelhans) [inline collectors/python.d.plugin/unbound/README.md]: I don't think this file is supposed to exist any more! Not sure how it ended up here. @shortpatti, I think you can delete this on your branch.

**Comment by joelhans** [comment]
Hey @ilyam8—so, to be clear, let's use an example.

Instead of **# NUT monitoring with Netdata**, you would prefer the header say **# UPS/PDU monitoring with Netdata"?

File: collectors/python.d.plugin/go_expvar/README.md [hunk 6]
```diff
@@ -1,4 +1,4 @@
-# go_expvar
+# Go Expvar monitoring with Netdata
```
Comment (joelhans) [inline collectors/python.d.plugin/go_expvar/README.md]:
```suggestion
# Go application monitoring with Netdata
```

File: collectors/python.d.plugin/hddtemp/README.md [hunk 7]
```diff
@@ -1,6 +1,6 @@
-# hddtemp
+# HDD Temperature monitoring with Netdata
```
Comment (joelhans) [inline collectors/python.d.plugin/hddtemp/README.md]:
```suggestion
# Hard drive temperature monitoring with Netdata
```

File: collectors/python.d.plugin/httpcheck/README.md [hunk 8]
```diff
@@ -1,6 +1,6 @@
-# httpcheck
+# Http Check monitoring with Netdata
```
Comment (joelhans) [inline collectors/python.d.plugin/httpcheck/README.md]:
```suggestion
# Remote web server monitoring with Netdata
```

File: collectors/python.d.plugin/portcheck/README.md [hunk 9]
```diff
@@ -1,6 +1,6 @@
-# portcheck
+# PortCheck monitoring with Netdata
```
Comment (joelhans) [inline collectors/python.d.plugin/portcheck/README.md]: @ilyam8 What would you recommend here?

Comment (ilyam8) [reply collectors/python.d.plugin/portcheck/README.md]: i would recomment follow `Monitors` column from https://docs.netdata.cloud/collectors/collectors/

Comment (ilyam8) [reply collectors/python.d.plugin/portcheck/README.md]: portcheck monitors any tcp endpoint

File: collectors/python.d.plugin/httpcheck/README.md (hunk 8 above)
Comment (ilyam8) [reply collectors/python.d.plugin/httpcheck/README.md]: Not web server, any HTTP endpoint

File: collectors/python.d.plugin/portcheck/README.md (hunk 9 above)
Comment (joelhans) [reply collectors/python.d.plugin/portcheck/README.md]: Okay, so **# TCP endpoint monitoring with Netdata**

Comment (shortpatti) [reply collectors/python.d.plugin/portcheck/README.md]: @ilyam8 I'll follow the Monitors column you suggested.  Thanks :)

File: collectors/python.d.plugin/unbound/README.md (hunk 5 above)
Comment (shortpatti) [reply collectors/python.d.plugin/unbound/README.md]: It's been removed :)

**Comment by ilyam8** [comment]
> nstead of # NUT monitoring with Netdata

see https://github.com/netdata/netdata/pull/8074#discussion_r378767983

I think we need to say what it monitors, then we dont use NUT, because module doesnt monitors nut

File: collectors/python.d.plugin/portcheck/README.md (hunk 9 above)
Comment (ilyam8) [reply collectors/python.d.plugin/portcheck/README.md]: thank you @shortpatti 👍

File: collectors/charts.d.plugin/libreswan/README.md (hunk 2 above)
Comment (shortpatti) [reply collectors/charts.d.plugin/libreswan/README.md]: **Libreswan IPSec tunnel monitoring with Netdata**

File: collectors/charts.d.plugin/sensors/README.md (hunk 4 above)
Comment (shortpatti) [reply collectors/charts.d.plugin/sensors/README.md]: **Network UPS tools monitoring with Netdata**

File: collectors/charts.d.plugin/nut/README.md (hunk 3 above)
Comment (shortpatti) [reply collectors/charts.d.plugin/nut/README.md]: **Network UPS tools monitoring with Netdata**

File: collectors/charts.d.plugin/sensors/README.md (hunk 4 above)
Comment (shortpatti) [reply collectors/charts.d.plugin/sensors/README.md]: **Linux machine sensors monitoring with Netdata**

File: collectors/python.d.plugin/web_log/README.md [hunk 10]
```diff
@@ -1,4 +1,6 @@
-# Web log monitoring with Netdata
+# Apache/NGINX/Squid monitoring with Netdata
```
Comment (joelhans) [inline collectors/python.d.plugin/web_log/README.md]: I do think we should mention that this is _log_ monitoring, and thus different from the Apache/Nginx modules.

```suggestion
# Apache/NGINX/Squid log monitoring with Netdata
```

File: collectors/python.d.plugin/dockerd/README.md [hunk 11]
```diff
@@ -1,6 +1,6 @@
-# Dockerd monitoring with Netdata
+# Docker engine monitoring with Netdata
```
Comment (joelhans) [inline collectors/python.d.plugin/dockerd/README.md]: I think, technically, engine should be capitalized.

```suggestion
# Docker Engine monitoring with Netdata
```

File: collectors/python.d.plugin/bind_rndc/README.md [hunk 12]
```diff
@@ -1,8 +1,8 @@
-# bind_rndc
+# Bind RNDC monitoring with Netdata
```
Comment (ilyam8) [inline collectors/python.d.plugin/bind_rndc/README.md]: BIND or ISC BIND

> https://www.isc.org/bind/

`rndc` is a tool which module uses to collect data.

File: collectors/python.d.plugin/isc_dhcpd/README.md [hunk 13]
```diff
@@ -1,8 +1,8 @@
-# isc_dhcpd
+# ISC DHCPD monitoring with Netdata
```
Comment (ilyam8) [inline collectors/python.d.plugin/isc_dhcpd/README.md]: ISC DHCP

> https://www.isc.org/dhcp/

dhcpd is daemon name

File: collectors/python.d.plugin/varnish/README.md [hunk 14]
```diff
@@ -1,8 +1,7 @@
-# varnish
+# Varnish cache monitoring with Netdata
```
Comment (ilyam8) [inline collectors/python.d.plugin/varnish/README.md]: I think it should be `Varnish Cache`

> https://varnish-cache.org/

File: collectors/python.d.plugin/adaptec_raid/README.md [hunk 15]
```diff
@@ -1,12 +1,18 @@
-# adaptec raid
+# Adaptec RAID controller monitoring with Netdata
 
-Module collects logical and physical devices health metrics.
+Collects health metrics on logical and physical devices.
```
Comment (ilyam8) [inline collectors/python.d.plugin/adaptec_raid/README.md]: Probably i dont translate it correctly, but the module collects _logical and physical devices metrics_, not _on logical and physical devices metrics_.

**Comment by ilyam8** [comment]
@shortpatti a request for the future - please do not force push if there is no real need, just commit. It is impossible to understand what was changed after force-push.

**Pull request merged**